IronFrame.Test/LocalTcpPortManagerTests.cs

```diff
@@ -130,5 +130,28 @@ public void ThrowWardenExecptionIfFirewallManagerFails()
                 Assert.IsType<Exception>(exception);
             }
         }
+
+        public class CreateFirewallRule : LocalTcpPortManagerTests
+        {
+            [Fact]
+            public void DelegatesToFirewallManager()
+            {
+                var tcpPortManager = new LocalTcpPortManager(FirewallManager, NetShRunner);
+                var firewallRuleSpec = Substitute.For<FirewallRuleSpec>();
```

Comment:
In this case the substitute is the same as calling `new FirewallRuleSpec()` because nothing on `FirewallRuleSpec` is virtual.

When I see `Substitute.For<T>` I expect to see specific behavior defined (or validated), but didn't in this case which was confusing.